### DISCUSSION TITLE
Autoplay

### DISCUSSION THREAD

**Discussion by ideiasfrescas** [videojs/video.js#7973] [Q&A]
Hello

I would like a small help regarding videojs misbehaving.
When I load the page, either after I type the url into Chrome's 105 address bar or when I press refresh ( via icon and or CTRL+F5 ) videojs does not trigger autoplay. 

After the page loaded, if I click a link to go to /, videojs autoplay triggers, despite i explicitly setup to do not autoplay and video tag has also autoplay attr to false. 
This is issue is also found on Edge, but not on Firefox...

I have setup videojs with the following code:

```js
var player = videojs(document.querySelector('#my-video'), {
    controls: true,
    autoplay: false,
    inactivityTimeout: 1000*5,
    preload: 'auto',
    fluid: true,
})
 //event trapping code follows
```

Have you encountered a similiar issue?

**Comment by gkatsev** [comment]
Have a minimal live example for this? We have not seen erroneous autoplays before.

**Comment by densen2014** [comment]
Try this: 

because new rule of browse is:  if you want to autoplay , you need muted your player.

```
    <video id="@Id"
           class="video-js"
           muted
           webkit-playsinline
           playsinline
           x-webkit-airplay="allow"
           x5-video-player-type="h5"
           x5-video-player-fullscreen="true"
           x5-video-orientation="portraint">
        <p class="vjs-no-js">
            To view this video please enable JavaScript, and consider upgrading to a
            web browser that
            <a href="https://videojs.com/html5-video-support/" target="_blank">
                supports HTML5 video
            </a>
        </p>
    </video>
```
